app/src/main/java/com/release/keyneez/presentation/main/search/SearchAdapter.kt

```diff
@@ -0,0 +1,35 @@
+package com.release.keyneez.presentation.main.search
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.release.keyneez.data.entity.response.ResponseGetSearchResultDto
+import com.release.keyneez.databinding.ItemSearchContentBinding
+
+class SearchAdapter : RecyclerView.Adapter<SearchAdapter.getViewHolder>() {
+
+    var data = listOf<ResponseGetSearchResultDto>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): getViewHolder {
+        val binding =
+            ItemSearchContentBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return getViewHolder(binding)
+    }
+
+    override fun getItemCount(): Int = data.size
+
+    override fun onBindViewHolder(holder: getViewHolder, position: Int) {
+        holder.bind(data[position])
+    }
+
+    class getViewHolder(private val binding: ItemSearchContentBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun bind(item: ResponseGetSearchResultDto) {
+            binding.root.setOnClickListener {
```

Comment:
```suggestion
            binding.root.setOnSingleClickListener {
```
아직 제대로 구현 안 하신 것 같지만 SingleClickListener를 사용해주세요!